README.md

```diff
@@ -1,30 +1,104 @@
+[![Go Report Card](https://goreportcard.com/badge/softonic/node-policy-webhook)](https://goreportcard.com/report/softonic/node-policy-webhook)
+[![Releases](https://img.shields.io/github/release-pre/softonic/node-policy-webhook.svg?sort=semver)](https://github.com/softonic/node-policy-webhook/releases)
+[![LICENSE](https://img.shields.io/github/license/softonic/node-policy-webhook.svg)](https://github.com/softonic/node-policy-webhook/blob/master/LICENSE)
+[![DockerHub](https://img.shields.io/docker/pulls/softonic/node-policy-webhook.svg)](https://hub.docker.com/r/softonic/node-policy-webhook)
+
+
 # node-policy-webhook
 K8s webhook handling profiles for tolerations, nodeSelector and nodeAffinity
 
+# Quick Start
+
+
+Steps to got webhook working
+
+- Deploy the webhook ( kubectl or helm )
+- Deploy NodePolicyProfile object
+- Deploy with the annotation ( below you can find it )  and watch if the pod is scheduled where was meant to
+
+## Deployment
 
-## DEVEL ENVIRONMENT
 
-### Requirements
+### Install k8s platform
 
-Install kind
+In this example we will use kind, but you can use whatever k8s platform you are comfortable with
 
 ```bash
 curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-$\(uname\)-amd64
 mv kind-darwin-amd64 /usr/local/bin/kind
+kind create cluster
 ```
 
+### Deploy using kubectl 
+
+
+Set the variables 
 
 ```bash
+VERSION ?= 0.1.1
```

Comment:
No need to set this in Quick Start mode. Just get whatever there is. We can move this to a more in-depth section, what do you think?